__tests__/app.test.js

```diff
@@ -0,0 +1,35 @@
+const app = require("../app");
+const request = require("supertest");
+const data = require("../db/data/test-data/index.js");
+const db = require("../db/connection.js");
+const seed = require("../db/seeds/seed.js");
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+describe("404: Endpoint not found", () => {
+  test("should trigger a catch all endpoint for any requests that arent found in the api, and return a 404", () => {
+    return request(app)
+      .get("/api/wrongendpoint")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("The requested endpoint does not exist.");
+      });
+  });
+});
```

Comment:
Good to see that you have extracted this out into it's own endpoint, as it's not specific to topics. 👍